haystack/nodes/audio/answer_to_speech.py

```diff
@@ -49,9 +52,13 @@ def __init__(
                 By default, the audio file gets the name from the MD5 sum of the input text.
         :param transformers_params: The parameters to pass over to the `Text2Speech.from_pretrained()` call.
         :param progress_bar: Whether to show a progress bar while converting the text to audio.
+        :param devices: List of GPU devices to limit inference to certain GPUs and not use all available ones (e.g. [torch.device('cuda:0')]).
```

Comment:
Does it mean the following:
List of GPU devices you want to limit inference to so as not to use all available GPUs. Example: [torch.device(cuda:0)].